app/views/layouts/application.html.erb

```diff
@@ -14,19 +14,16 @@
       <% if session[:current_user_id] %>
         <%= render  partial: "/layouts/sidebar" %>
       <% end %>
-      <% if flash[:error] %>
-        <%= render partial: "/layouts/flash-message" %>
-      <% end %>
-      <% if flash[:notice] %>
-        <%= render partial: "/layouts/flash-message" %>
-      <% end %>
       <div class="flex flex-col w-0 flex-1 overflow-hidden">
         <main
           class="flex-1 relative z-0 overflow-y-auto focus:outline-none"
           tabindex="0"
         >
           <div class="py-6">
             <div class="max-w-7xl mx-auto px-4 sm:px-6 md:px-8 mt-10 md:mt-0">
+              <% if flash[:error] || flash[:notice] %>
+                <%= render partial: "/layouts/flash-message" %>
+              <% end %>
               <%= yield %>
```

Comment:
Improve Signup page view. The signup page is shifted down in my screen
![image](https://user-images.githubusercontent.com/52771399/114274960-5d5da480-9a3e-11eb-98bb-43e38be4cbfe.png)

Reply:
Done 🚀